spotify-app/src/CurrentlyPlaying.re

```diff
@@ -62,36 +71,56 @@ let make =
   render: self =>
     ReasonReact.(
       <div className="current-playing">
-        <div className=playerWrapper>
-          <img className=albumImage src=albumImageUrl alt="Album Image" />
-          <div
-            className={SharedCss.flexWrapper(~justify=`spaceBetween, ~align=`center)}>
-            <div>
-              <h3 className=songNameStyle> {string(songName)} </h3>
-              <p className=artistNameStyle> {string(artistName)} </p>
-            </div>
-            {
-              isPlaying ?
-                null :
-                <img className=pauseBtnStyle src=pause alt="Pause Icon" />
-            }
-            <SoundWave isPlaying />
-          </div>
-          <div
-            className={
-              Cn.make([
-                SharedCss.flexWrapper(~justify=`spaceBetween, ~align=`center),
-              ])
-            }>
-            <Progress
-              color="success"
-              className=progressBarStyle
-              value=progressPct
+
+          <div className={Cn.make([playerWrapper])}>
+            <img
+              key=songName
```

Comment:
are you using a key here to trigger the animation again when the song changes? :100: